test/box/tx_man.test.lua

```diff
@@ -1182,6 +1182,45 @@ box.execute([[DROP TABLE k3;]])
 box.execute([[DROP TABLE k2;]])
 box.execute([[DROP TABLE k1;]])
 
+-- https://github.com/tarantool/tarantool/issues/6396
+
+s = box.schema.create_space('test')
+pk = s:create_index('pk', {parts={1, 'uint'}})
+sk = s:create_index('sk', {parts={2, 'uint'}})
```

Comment:
why do you need to create smth to test `box.txn_id()`?

Reply:
Moreover I guess tx_man.test.lua actually doesn't seem to be suitable for this test. Transaction ID is not only about mvcc, it is also available in casual TX mode